src/utils/validation.ts

```diff
@@ -57,16 +57,23 @@ export const addressIsNotCurrentSafe =
 
 export const FLOAT_REGEX = /^[0-9]+([,.][0-9]+)?$/
 
-export const validateTokenAmount = (amount: string, token?: { balance: string; tokenInfo: TokenInfo }) => {
-  if (!token) return
-
+export const validateNumber = (amount: string) => {
```

Comment:
Not a very descriptive name. `validateAmount` would fit more.

Reply:
There is already a `validateAmount` in this file.

Reply:
Please find suitable names for both of them then. `validateNumber` is no bueno, because it doesn't pass just any number but only positive ones and it says "amount" in the error message.